interface/src/lib/components/dialogs/orderConfirm.svelte

```diff
@@ -66,31 +78,45 @@
       }
     }
     const trace = getUuid();
-    
-    SpotPay({ $orderTypeLimit, buy: $buy, symbol: $pair.symbol, exchange: $pair.exchange, price: String($limitPrice), amount: payAmount, trace})
 
-    
+    SpotPay({
+      limit: $orderTypeLimit,
+      buy: $buy,
+      symbol: $pair.symbol,
+      exchange: $pair.exchange,
+      price: String($limitPrice),
+      amount: payAmount ? String(payAmount * 1.2) : '',
+      trace
+    })
+
+
     let found = false;
     let totalTime = 0;
 
     var interval = setInterval(async () => {
+      if (!$orderConfirmDialog) {
+        clearInterval(interval);
+      }
+
       console.log(`${new Date()} called`);
```

Comment:
Is this console.log necessary?

Reply:
@leric7 
I've removed this one console.log from the code. It wasn't added by me. And there are still a lot more in the codebase that you might want to remove.

Reply:
@leric7 
I've removed this one console.log from the code. It wasn't added by me. And there are still a lot more in the codebase that you might want to remove.